apps/stupidchat/stupid_chat.star

```diff
@@ -0,0 +1,81 @@
+"""
+Applet: Stupid Chat
+Summary: Tidbyt Messaging
+Description: Send messages to your Tidbyt via https://stupid.chat or an API.
+Author: harrisonpage
+"""
+
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+
+VERSION = "1.0.0"
+
+def get_config(config, key, default):
+    value = config.get(key)
+    if not value:
+        value = default
+    return value
+
+def main(config):
+    token = get_config(config, "token", "WELCOME")
+    client_id = get_config(config, "client_id", "XXX")
```

Comment:
I take it this is for debug purposes? Just wanted to mention since it's not in the schema.